test/suites/integration/validation_test.go

```diff
@@ -103,13 +103,13 @@ var _ = Describe("Validation", func() {
 		})
 		It("should error when ttlSecondAfterEmpty is negative", func() {
```

Comment:
Seems like we should change the name of this testCase now that this field isn't called this anymore